### ISSUE TITLE
datasets generator   program

### ISSUE THREAD

**Issue by wuxiangrui188** [atenpas/gpd#116]
First,thank you for your amazing work!
could you send the program for generate  datasetes(like the bigbird datasets) ? Or samething like the datasets file construct?
In my opinion ,the datasets structure should be include:(example 1 object)
0_gt.pcd   0_gt.csv
0_1.pcd 0_2.pcd ......0_20.pcd (its corresponding .csv file for store normal data)

it is that true?

hope for your helpping!

**Comment by zhoumiaoqiang** [comment]
> First,thank you for your amazing work! could you send the program for generate datasetes(like the bigbird datasets) ? Or samething like the datasets file construct? In my opinion ,the datasets structure should be include:(example 1 object) 0_gt.pcd 0_gt.csv 0_1.pcd 0_2.pcd ......0_20.pcd (its corresponding .csv file for store normal data)
> 
> it is that true?
> 
> hope for your helpping!

Hello, I am also confused about this. Have you completed the generation of the data set?

**Comment by TechyRio** [comment]
> > First,thank you for your amazing work! could you send the program for generate datasetes(like the bigbird datasets) ? Or samething like the datasets file construct? In my opinion ,the datasets structure should be include:(example 1 object) 0_gt.pcd 0_gt.csv 0_1.pcd 0_2.pcd ......0_20.pcd (its corresponding .csv file for store normal data)
> > it is that true?
> > hope for your helpping!
> 
> Hello, I am also confused about this. Have you completed the generation of the data set?

yes,i just use the function in the program of other files to generate the Normal vectors. you can try again!

**Comment by zhoumiaoqiang** [comment]
> yes,i just use the function in the program of other files to generate the Normal vectors. you can try again!

Can you add a contact number?QQ783145004

**Comment by huiwenzhang** [comment]
> 

@TechyRio  The data format is exactly as you pasted above according to the code. However, the bigbird dataset format is very different from the format used in this repo. Which program did you use to generate the dataset? Could U explain it more clearly?

**Comment by huiwenzhang** [comment]
@qyp-robot   which c++ file you used for generating data? Basically, U don't need two cameras. The full mesh model is used to evaluate valid antipodal grasps, i.e. the label for sampled grasps. U can't train with full mesh model, except that you can get the full mesh when doing inference in real world.
